### PR TITLE
feature image + small css fix

### PR DESCRIPTION
**Pull request by tomaszrondio**

### CODE REVIEW

**Review by ahilles107 — CHANGES_REQUESTED** [review]
Please fix your editor issues with tabs and spaces. We use 4 spaces per tab. Fix file to use always spaces.

**Comment by tomaszrondio** [comment]
in themes we always used 2 spaces per tab. Having 4 would require rewriting whole theme.

**Comment by ahilles107** [comment]
then ok, it can be 2 spaces, but use allway spaces not mixed with tabs like it's now.

File: views/article.html.twig [hunk 1]
```diff
@@ -18,17 +18,17 @@
...
-		  {% if media.renditions is iterable %}
-		    {% for rendition in media.renditions %}
-
-		      {% if rendition.name == '1000x1000' %}
-		        	data-lg="{{path(rendition)}}"
-		      {% endif %}
-		    {% endfor %}
-		  {% endif %}
-		{% endgimmelist %}
+			{% if gimme.article.featureMedia %}
```
Comment (ahilles107) [inline views/article.html.twig]: use spaces instead tabs

**Review by takeit — APPROVED** [review]

**Pull request merged**